### PR TITLE
Add datafile REST API info to server-side docs

### PR DESCRIPTION
**Pull request by delikat**
attn @jprovine

### CODE REVIEW

File: src/content/server/reference/datafile/0-intro.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ This datafile compactly represents all of the "instructions" needed to activate
 
 You can fetch the datafile for your Optimizely project in two ways:
 
-* *Use the Optimizely CDN.*  For example, if the ID of your project is `12345` you can access the file at http://cdn.optimizely.com/json/12345.json.
-* *Use the Optimizely REST API.* (Details TBD)
+* *Use the Optimizely CDN.*  For example, if the ID of your project is `12345` you can access the file at https://cdn.optimizely.com/json/12345.json.
+* *Use the Optimizely REST API.* For example, if the ID of your proejct is `12345` you can access the file at https://www.optimizelyapis.com/experiment/v1/projects/12345/json. Please note that as with other requests to the REST API, you will have to [authenticate with an API token](/rest/getting-started).
```
Comment (haleybash-optimizely) [inline src/content/server/reference/datafile/0-intro.md]: Quick nit: `project` is misspelled (ID of your project)

Comment (aliabbasrizvi) [inline src/content/server/reference/datafile/0-intro.md]: nit. proejct is misspelled.

**Comment by aliabbasrizvi** [comment]
LGTM. Please merge and deploy after fixing the spelling error.

**Comment by delikat** [comment]
O_O thanks for review.

**Pull request merged**